Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.20@sha256:f7099345b8e4a93c62dc5102e7eb19a9cdbad12e7e322644eeaba355d70e616d
+FROM 445615400570.dkr.ecr.us-east-1.amazonaws.com/ecr-public/docker/library/golang:1.20
```

Comment:
We should keep the digest if we can, to ensure the image doesn't change if upstream mutates the tag.